From 746fa830c6d7d302a3e465054143fd3b3a130367 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 13 Oct 2021 12:37:59 +0200
Subject: [PATCH] python-zipp: pin setuptools-scm version

The recent version of setuptools-scm depends on tomli, which has some build issues.
Older one works.

Suggested-by: Jeffery To <jeffery.to@gmail.com>
Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/python/python-zipp/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/python/python-zipp/Makefile b/lang/python/python-zipp/Makefile
index 69a7ebed1..138211471 100644
--- a/lang/python/python-zipp/Makefile
+++ b/lang/python/python-zipp/Makefile
@@ -25,7 +25,7 @@ define Package/python3-zipp
   VARIANT:=python3
 endef
 
-HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=setuptools-scm
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:="setuptools-scm==5.0.2"
 
 define Package/python3-zipp/description
   Backport of pathlib-compatible object wrapper for zip files
-- 
2.30.2

